scss/components/_settings-panel.scss

```diff
@@ -0,0 +1,16 @@
+.settings-panel {
+
+  .error-box {
+    max-width: 80rem;
+    margin-bottom: 2rem;
+  }
+
+  fieldset .fieldset-input {
```

Comment:
I'd remove this, along with the wrapping elements, and replace it with this in the `fieldset` styles:

```
fieldset {
  > {
    *:not(:last-child) {
      margin-bottom: 1rem;
    }

    input,
    textarea,
    .validated-input {
      margin-left: 1rem;
    }
  }
}
```